README.md

```diff
@@ -83,311 +83,391 @@ spec:
+      baseImageUrl: http://host/images/rhel-image-8.4.x86_64.qcow2
+      # NOTE: these are automatically created via the OpenStackControlplane CR above
+      deploymentSSHSecret: osp-controlplane-ssh-keys
+      # The interface on the nodes that will be assigned an IP from the mgmtCidr
+      ctlplaneInterface: enp2s0
+      # Networks to associate with this host
       networks:
         - ctlplane
-      cores: 6
-      memory: 12
-      diskSize: 40
-      baseImageURL: http://host/images/rhel-guest-image-8.3-417.x86_64.qcow2
-      storageClass: host-nfs-storageclass
-```
-
-If you write the above YAML into a file called openstackcontrolplane.yaml you can create the OpenStackControlPlane via this command:
-
-```bash
-oc create -f openstackcontrolplane.yaml
-```
-
-5) Define an OpenStackBaremetalSet to scale out OSP Compute hosts. The OpenStackBaremetal resource can be used to define and scale Compute resources and optionally be used to define and scale out baremetal hosts for other types of TripleO roles. The example below defines a single Compute host to be created.
-
-Note: If the rhel-guest-image is used as base to deploy the OpenStackBaremetalSet compute nodes, make sure to remove the net.ifnames=0 kernel parameter from the image to have the biosdev network interface naming. This can be done like:
-
-```bash
-dnf install -y libguestfs-tools-c
-virt-customize -a bms-image.qcow2 --run-command 'sed -i -e "s/^\(kernelopts=.*\)net.ifnames=0 \(.*\)/\1\2/" /boot/grub2/grubenv'
-```
-
-```yaml
-apiVersion: osp-director.openstack.org/v1beta1
-kind: OpenStackBaremetalSet
-metadata:
-  name: compute
-  namespace: openstack
-spec:
-  # How many nodes to provision
-  count: {{ osp_compute_count }}
-  # The image to install on the provisioned nodes. NOTE: needs to be accessible on the OpenShift Metal3 provisioning network.
-  baseImageUrl: http://host/images/rhel-image-8.3-417.x86_64.qcow2
-  # NOTE: these are automatically created via the OpenStackControlplane CR above
-  deploymentSSHSecret: osp-controlplane-ssh-keys
-  # The interface on the nodes that will be assigned an IP from the mgmtCidr
-  ctlplaneInterface: enp2s0
-  # Networks to associate with this host
-  networks:
-    - ctlplane
-  roleName: Compute
-  passwordSecret: userpassword
-```
+        - internalapi
+        - tenant
+        - storage
+      roleName: Compute
+      passwordSecret: userpassword
+    ```
 
-If you write the above YAML into a file called compute.yaml you can create the OpenStackBaremetalSet via this command:
+    If you write the above YAML into a file called compute.yaml you can create the OpenStackBaremetalSet via this command:
 
-```bash
-oc create -f compute.yaml
-```
+    ```bash
+    oc create -f compute.yaml
+    ```
 
 6) Wait for any resources virtual machine or baremetal host resources to finish provisioning. FIXME: need to add status info to our CR's so that we can add wait conditions on these resources.
```

Comment:
We should be able to wait on these resources now, right?  Since we added the `Condition` to the statuses?

Reply:
I need to update this. this is no longer required as we already wait in the osplaybookgenerator for osbms and osvmset be ready. https://github.com/openstack-k8s-operators/osp-director-operator/pull/271

Reply:
removed step 6